docs/user/auth-resources.md

```diff
@@ -0,0 +1,56 @@
+# Authentication resources
+
+The table below contains the routes, names, resource names, and verbs for flightctl API endpoints:
+
+|Route| Name| Resource| Verb |
+|-----|-----|---------|------|
+|`DELETE /api/v1/certificatesigningrequests`|`DeleteCertificateSigningRequests`|`certificatessigningrequests`|`deletecollection`|
+|`GET /api/v1/certificatesigningrequests`|`ListCertificateSigningRequests`|`certificatessigningrequests`|`list`|
+|`POST /api/v1/certificatesigningrequests`|`CreateCertificateSigningRequest`|`certificatessigningrequests`|`create`|
+|`DELETE /api/v1/certificatesigningrequests/{name}`|`DeleteCertificateSigningRequest`|`certificatessigningrequests`|`delete`|
+|`GET /api/v1/certificatesigningrequests/{name}`|`ReadCertificateSigningRequest`|`certificatessigningrequests`|`read`|
+|`PATCH /api/v1/certificatesigningrequests/{name}`|`PatchCertificateSigningRequest`|`certificatessigningrequests`|`patch`|
+|`PUT /api/v1/certificatesigningrequests/{name}`|`ReplaceCertificateSigningRequest`|`certificatessigningrequests`|`replace`|
+|`DELETE /api/v1/certificatesigningrequests/{name}/approval`|`DenyCertificateSigningRequest`|`certificatessigningrequests`|`deny`|
+|`POST /api/v1/devices`|`CreateDevice`|`devices`|`create`|
+|`GET /api/v1/devices`|`ListDevices`|`devices`|`list`|
+|`DELETE /api/v1/devices`|`DeleteDevices`|`devices`|`deletecollection`|
+|`GET /api/v1/devices/{name}`|`ReadDevice`|`devices`|`read`|
+|`PUT /api/v1/devices/{name}`|`ReplaceDevice`|`devices`|`replace`|
+|`DELETE /api/v1/devices/{name}`|`DeleteDevice`|`devices`|`delete`|
+|`GET /api/v1/devices/{name}/status`|`ReadDeviceStatus`|`devices`|`readstatus`|
+|`PUT /api/v1/devices/{name}/status`|`ReplaceDeviceStatus`|`devices`|`replacestatus`|
```

Comment:
```suggestion
|`GET /api/v1/devices/{name}/status`|`ReadDeviceStatus`|`devices/status`|`get`|
|`PUT /api/v1/devices/{name}/status`|`ReplaceDeviceStatus`|`devices/status`|`update`|
```

Reply:
The resource being read/updated is the device.  Do we want to have different resource?
Can a resource contain a '/' ? I don't recall that a resource in CRD can contain a '/'.

Reply:
The resource name is just a string, we can use whatever we want. The point is that the authorization for the status must be different, i.e. we need to allow updating status without being able to update spec, for example.